spec/unit/mixin/shell_out_spec.rb

```diff
@@ -18,20 +18,42 @@
 
 require "spec_helper"
 
-describe Ohai::Mixin::Command, "shell_out" do
+describe Ohai::Mixin::ShellOut, "shell_out" do
   let(:cmd) { "sparkle-dream --version" }
 
-  let(:shell_out) { double("Mixlib::ShellOut") }
+  let(:shell_out) { double("Mixlib::ShellOut", live_stream: nil, :live_stream= => nil) }
 
   let(:plugin_name) { :OSSparkleDream }
 
-  let(:options) { windows? ? { timeout: 30 } : { timeout: 30, env: { "PATH" => "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" } } }
+  let(:timeout) { 30 }
+
+  let(:options) do
+    if windows?
+      { timeout: timeout }
+    else
+      {
+        timeout: timeout,
+        environment: {
+          "LANG" => "en_US.UTF-8",
+          "LANGUAGE" => "en_US.UTF-8",
+          "LC_ALL" => "en_US.UTF-8",
+          "PATH" => "/Users/lamont/.asdf/installs/ruby/2.7.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
```

Comment:
yeah after discussing on slack this seems more of a bugfix and correct behavior, since it forces output of utilities to a consistent locale for parsing the output of utilties as chef-client itself has done for about 7-8 years now.